### ISSUE TITLE
Can't hide titlebar with x11_hide_window_decorations

### ISSUE THREAD

**Issue by GKFX** [kovidgoyal/kitty#824]
Below is the output of `kitty --debug-config`:

```
kitty 0.11.3 created by Kovid Goyal
Linux gbat-hp-linux 4.17.0-1-amd64 #1 SMP Debian 4.17.8-1 (2018-07-20) x86_64
Running under: X11
Debian GNU/Linux buster/sid \n \l
Loaded config files: /home/bateman/.config/kitty/kitty.conf

Config options different from defaults:
active_tab_background       Color(red=187, green=187, blue=187)
active_tab_font_style       (True, False)
font_size                   12.0
foreground                  Color(red=255, green=255, blue=255)
inactive_tab_background     Color(red=136, green=136, blue=136)
inactive_tab_foreground     Color(red=0, green=0, blue=0)
tab_bar_style               separator
tab_separator                ▅ 
x11_hide_window_decorations True
```
So unless I'm being daft, that `x11_hide_window_decorations` setting should be sufficient to hide the titlebar. But it doesn't seem to, even when it was the only setting I had. I've tried closing and re-opening Kitty, and I've rebooted too since initially writing this config file. The other settings have all taken effect. 
I'm using Gnome 3 if that helps.
Many thanks for creating Kitty!

**Comment by kovidgoyal** [comment]
As is noted in the help for that option, whether it works or not depends on whether your window manager supports it. All kitty does is mark the window the the appropriate hint, then it is up to the window manager to honor it or not. If it does not, you may need to configure the window manager witha special rule to hide the titlebar for windows with WM_CLASS kitty.

**Comment by GKFX** [comment]
Since Gnome 3 is a very common system, if I did some investigating and wrote a patch to make the option work out-of-the-box would you be likely to accept it?

**Comment by kovidgoyal** [comment]
Depends on how invasive the patch is, but in principle, sure.

**Comment by GKFX** [comment]
Turns out you've already fixed it at some point after the last release!

**Comment by kovidgoyal** [comment]
Good to know :) There are actually a lot of fixes pending for 0.12 -- I just dont have the bandwidth right now to make a release.
